eclair-core/src/test/scala/fr/acinq/eclair/channel/states/b/WaitForFundingCreatedInternalStateSpec.scala

```diff
@@ -68,7 +68,10 @@ class WaitForFundingCreatedInternalStateSpec extends TestKitBaseClass with Fixtu
 
   test("recv CMD_CLOSE") { f =>
     import f._
-    alice ! CMD_CLOSE(None)
+    val sender = TestProbe()
+    val c = CMD_CLOSE(sender.ref, None)
```

Comment:
Should we have one of these tests test the fallback to `sender` when `ActorRef.noSender` is provided?

```scala
val sender = TestProbe()
val c = CMD_CLOSE(ActorRef.noSender, None)
sender.send(alice, c)
sender.expectMsg(RES_SUCCESS(c, ByteVector32.Zeroes))
awaitCond(alice.stateName == CLOSED)
```

Reply:
Done with 2bad181ab... and found a bug!